XavierGuertin/WildfireVisualizationProject
# #47 - feat(frontend) - Added Unit Tests and Toggleability to the Metadata

### Summary
Added the ability to toggle the metadata along with adding unit tests for the component

### Changes Made
- Added unit tests
- Added ability to toggle component
- Moved away from StyledComponents into a css sheet

### Testing Instructions

1. Run the frontend
2. Select a dataset from the available datasets
3. Once loaded, click on the header of the metadata
4. The component should collapse
5. Press on the collapsed component
6. The metadata component should open back up
